go/pkg/cas/client.go

```diff
@@ -28,12 +28,32 @@ type Client struct {
 
 	byteStream bspb.ByteStreamClient
 	cas        repb.ContentAddressableStorageClient
+
+	// Mockable functions.
+
+	testScheduleCheck func(ctx context.Context, item *uploadItem) error
 }
 
 // ClientConfig is a config for Client.
 // See DefaultClientConfig() for the default values.
 type ClientConfig struct {
-	// TODO(nodir): add conifg values here.
+	// Maximum number of concurrent file system operations.
```

Comment:
better to follow
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
?

Reply:
Done